rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/Authority.java

```diff
@@ -16,7 +12,10 @@
 import javax.persistence.ManyToMany;
 import javax.persistence.Table;
 import javax.validation.constraints.NotNull;
-import java.util.List;
+import lombok.AllArgsConstructor;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'lombok\.AllArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.